test-it/karma/utilities/TestHelpers.SpyHelpers.js

```diff
@@ -0,0 +1,17 @@
+(function () {
+    'use strict';
+
+    window.testHelpers = window.testHelpers || {};
```

Comment:
What does this line do?

Reply:
Basically the || operator works on truthy or falsy values. Depending on the order in which TestHelpers are loaded (or any other script) it is possible that this is the first one loaded, in that case window.testHelpers is `undefined` which is considered a falsy value. Therefore this line is saying, window.testHelpers is either an object (truthy) already defined by another script which loaded first, or is an empty object.